QUESTIONS.rst

```diff
@@ -5,20 +5,19 @@ To use the package template run ``cookiecutter gh:astropy/package-template``,
 once you have run this command you will be asked a series of questions. Below is
 a description of each of the prompts.
 
-1. ``package_name``: This is a human readable name for your package, like 'Astropy' or 'SunPy'.
-2. ``module_name``: This is the name of your python package i.e. 'astropy' or 'sunpy'.
+1. ``package_name``: This is a human readable name for your package, like ``Astropy`` or ``SunPy``.
+2. ``module_name``: This is the name of your python package i.e. ``astropy`` or ``sunpy``.
 3. ``short_description``: This is a one sentence description of your package.
-4. ``long_description``: This is a multi-line description of your package.
```

Comment:
I think twine chokes if the README is not valid rst, but otherwise I've never seen issues with complexity. I do think we want to encourage this approach since it produces much nicer pages on PyPI.